### ISSUE TITLE
ValueError: cannot convert float NaN to integer

### ISSUE THREAD

**Issue by Boring-Mind** [Kozea/WeasyPrint#1359]
I tried to convert html file to pdf using write_pdf function. You can find my html (variable content in code) and css (variable css1) files here: [html_and_css.zip](https://github.com/Kozea/WeasyPrint/files/6507544/html_and_css.zip)

My code:
``` python
font_config = FontConfiguration()
html = HTML(string=content)
css1 = CSS(string=css[0], font_config=font_config)
html.write_pdf("/path/to/some_pdf.pdf"),
stylesheets=[css1],
font_config=font_config,
)
```

Traceback:
```
File "/home/username/dev/legitcheck/server/guides/scrapers/guides_to_pdf_weasyprint.py", line 40, in convert_guide_to_pdf
    html.write_pdf(
  File "/home/username/.local/share/virtualenvs/server--FOsldmU/lib/python3.8/site-packages/weasyprint/__init__.py", line 222, in write_pdf
    self.render(
  File "/home/username/.local/share/virtualenvs/server--FOsldmU/lib/python3.8/site-packages/weasyprint/document.py", line 655, in write_pdf
    self.add_hyperlinks(links, anchors, context, scale)
  File "/home/username/.local/share/virtualenvs/server--FOsldmU/lib/python3.8/site-packages/weasyprint/document.py", line 576, in add_hyperlinks
    [int(round(i * scale)) for i in rectangle] +
  File "/home/username/.local/share/virtualenvs/server--FOsldmU/lib/python3.8/site-packages/weasyprint/document.py", line 576, in <listcomp>
    [int(round(i * scale)) for i in rectangle] +
ValueError: cannot convert float NaN to integer
```

Please let me know if you need any additional information about that issue.

**Comment by liZe** [comment]
Hello!

Thank you for this bug report. I can reproduce the problem with the current stable version (52.5) but I think that it’s fixed on the current master branch. Is it possible for you to test the master branch?

**Comment by Boring-Mind** [comment]
Hello! Yeah, I'll try it. Thank you for the quick response!

**Comment by Boring-Mind** [comment]
I didn't found any instructions how to run it from source. Maybe you have ones? I just don't want to bother you. Thank you in advance!

**Comment by liZe** [comment]
You don’t bother me!

You can install from git using `pip install git+https://github.com/Kozea/WeasyPrint`.

**Comment by Boring-Mind** [comment]
Thank you! I'll try it!

**Comment by Boring-Mind** [comment]
Yeah, it is fixed in master branch. But now I can't import weasyprint.fonts.FontConfiguration. Is that okay? I suppose that in a new version font configuration won't be needed.

**Comment by liZe** [comment]
> Yeah, it is fixed in master branch. But now I can't import weasyprint.fonts.FontConfiguration. Is that okay? I suppose that in a new version font configuration won't be needed.

It’s actually needed in the new version, but the modules have changed. You have to import `weasyprint.text.fonts.FontConfiguration` now. The [documentation](https://doc.courtbouillon.org/weasyprint/latest/first_steps.html#instantiating-html-and-css-objects) has been updated, but a line in the changelog could definitely help.
